### PR TITLE
Update board make file to use OV7692 driver as default.

### PR DESCRIPTION
**Pull request by aniktash**

### CODE REVIEW

**Review by vicloginov — COMMENTED** [review]
Looks good

File: Examples/MAX78000/CNN/cats-dogs_demo/README.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ BOARD=FTHR_RevA
 endif
 ```
 
+**Note: If you are using Eclipse, please also make sure to change the value of Board environment variable to "FTHR_RevA by:**
+
+*Right click project name > Properties > C/C++ Build > Environment > Board"*
+
+<img src="Resources\eclipse_board.png" style="zoom:33%;" />
```
Comment (rotx-maxim) [inline Examples/MAX78000/CNN/cats-dogs_demo/README.md:56]: in HTML, this needs to be a forward slash

File: Examples/MAX78000/CNN/cats-dogs_demo/README.md [hunk 2]
```diff
@@ -104,11 +112,11 @@ endif
 While using TFT display keep its power switch in "ON" position. The TFT "Reset" button also can be used as Feather reset.
 Press PB1 (SW1) button to start demo.
 
-<img src="/fthr_tft.png" style="zoom: 50%;" />
+<img src="Resources\fthr_tft.png" style="zoom: 50%;" />
```
Comment (rotx-maxim) [inline Examples/MAX78000/CNN/cats-dogs_demo/README.md:115]: /

File: Examples/MAX78000/CNN/cats-dogs_demo/README.md [hunk 3]
```diff
@@ -130,7 +138,7 @@ To create your own header file follow these steps:
 
 1.  Navigate to Utility directory. $ cd Utility
 2.  Download cat or dog image in this directory.
-3.  Open 'rgb.py' file and change the name of the image file on line 8. im = (Image.open('./image\_filename.format')). Save the cahanges.
+3.  Open 'rgb.py' file and change the name of the image file on line 8. im = (Image.open('./image\_filename.format')). Save the changes.
```
Comment (rotx-maxim) [inline Examples/MAX78000/CNN/cats-dogs_demo/README.md:141]: In Python, we need os.path.join()

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
Please fix the path separator / \ issues.

File: Examples/MAX78000/CNN/cats-dogs_demo/README.md (hunk 3 above)
Comment (aniktash) [reply Examples/MAX78000/CNN/cats-dogs_demo/README.md:141]: removed ./image, The step 2 instructs to copy to the same folder.

File: Examples/MAX78000/CNN/cats-dogs_demo/README.md [hunk 4]
```diff
@@ -130,7 +138,7 @@ To create your own header file follow these steps:
 
 1.  Navigate to Utility directory. $ cd Utility
 2.  Download cat or dog image in this directory.
-3.  Open 'rgb.py' file and change the name of the image file on line 8. im = (Image.open('./image\_filename.format')). Save the cahanges.
+3.  Open 'rgb.py' file and change the name of the image file on line 8. im = (Image.open('image\_filename.format')). Save the changes.
```
Comment (rotx-maxim) [inline Examples/MAX78000/CNN/cats-dogs_demo/README.md:141]: /

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
One more \

**Review by rotx-maxim — APPROVED** [review]

**Review by vicloginov — COMMENTED** [review]
Looks great!

**Pull request closed without merging**